.github/workflows/release_build.yml

```diff
@@ -0,0 +1,31 @@
+name: Release Go project
```

Comment:
I'm curious to hear from others, but I don' think we want this in this project. 

Also, why was `.github/workflows/docker.yml` disabled?